lib/mj-batik.js

```diff
@@ -0,0 +1,28 @@
+var fs = require('fs');
+var os = require('os');
+var path = require('path');
+var execFile = require('child_process').execFile;
+
+exports.svg2png = function(svg, result, scale, width, height, callback, check) {
+  var tmpfile = os.tmpdir() + "/mj-svg" +  process.pid;  // file name prefix to use for temp files
+  var BatikRasterizerPath = path.resolve(__dirname,'..','batik/batik-rasterizer.jar');
+  var batikCommands = ['-jar', BatikRasterizerPath, '-dpi', scale*72, tmpfile + '.svg'];
```

Comment:
If you want to use a width and height rather than dpi, batik can do that (according to the man page), so it would be possible to use those instead (and change or remove the command line dpi API, or base the scaling off the dpi, e.g., make scale = dpi / 96).  In that case, however, I'd only pass width and height and have the scale multiplied in before `svg2png` is called so that the `width` and `height` here are the true sizes that are being requested.